org.gnome.Papers.json

```diff
@@ -0,0 +1,181 @@
+{
+    "app-id": "org.gnome.Papers",
```

Comment:
`org.gnome` is reserved for official GNOME apps.

Is this an official GNOME app?

Reply:
The incubator submission

https://gitlab.gnome.org/GNOME/Incubator/Submission/-/issues/10

Seems to have been accepted.

Reply:
It's on the list https://gitlab.gnome.org/Teams/Releng/AppOrganization/-/blob/main/data/registered-app-ids.yml